agent/runner/runner_test.go

```diff
@@ -94,14 +98,6 @@ func TestCapacityLimit(t *testing.T) {
 
 	time.Sleep(time.Second)
 
-	// After second first job terminated and third job started
-	assert.False(t, cr.IsRunning(j1.ID()))
-	assert.True(t, cr.IsRunning(j2.ID()))
-	assert.True(t, cr.IsRunning(j3.ID()))
-	assert.False(t, cr.IsRunning(j4.ID()))
-
-	time.Sleep(time.Second)
-
```

Comment:
just curious why was this removed?

Reply:
Before there was one job task queue, I mean `j1` always started before `j2`, and `j2` before `j3`. With recent changes we don't have this type of behavior anymore, order depends on resources availability for each job.